components/todo-input.js

```diff
@@ -1,20 +1,72 @@
-import React from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
 
-export default function TodoInput() {
-  return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
-  );
+export default function TodoInput({ addTodo }) {
+    const [thisTodo, setThisTodo] = useState({
+    })
+
+    const inputChangeHandler = (el) => {
```

Comment:
1. 함수 이름을 지을때 동사가 맨 앞에 오도록 해주세요!
2. el이 무엇의 약자인가요..?
```suggestion
    const handleInputChange = (event) => {
```